songsuro/utils/util.py

```diff
@@ -1,6 +1,12 @@
 import re
 import string
+
 import torch
+import logging
+import sys
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+logger = logging
```

Comment:
??? I think this code is unfinished. Plus I think you have to move configuration of logger to the master `__init__.py`